packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-select.yaml

```diff
@@ -0,0 +1,21 @@
+Resources:
+  Subnet1:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId:
+        !Select [ 0, !GetAZs "" ]
+      CidrBlock: 10.0.0.0/24
+      AvailabilityZone:
+        !Select [ '0', !GetAZs eu-west-2 ]
+  Subnet2:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId:
+        !Select [ '0', !GetAZs "" ]
+      CidrBlock: 10.0.0.0/24
+      AvailabilityZone:
+        !Select [ 0, !GetAZs eu-west-2 ]
+  Bucket:
```

Comment:
I don't think we need this `Bucket` here all of a sudden... how about having `VpcId` of `Subnet2` as `Ref Subnet1`? I think we've already covered everything in `!Select [ '0', !GetAZs "" ]` in other parts of this template...